democracy/views/comment.py

```diff
@@ -91,8 +91,13 @@ def create(self, request, *args, **kwargs):
         if resp:
             return resp
 
+        # logged in users cannot set author_name
+        data = request.data.copy()
+        if request.user.is_authenticated():
+            data.pop('author_name', None)
```

Comment:
Maybe this should raise some sort of error instead of quietly doing things the user doesn't expect.